pipelines/facebook_ads/settings.py

```diff
@@ -0,0 +1,164 @@
+from typing import Any, Callable, Dict, Iterator, Literal
+
+from facebook_business.adobjects.abstractobject import AbstractObject
+
+TFbMethod = Callable[..., Iterator[AbstractObject]]
+
+
+DEFAULT_FIELDS = (
+    'id',
+    'updated_time',
+    'created_time',
+    'name',
+    'status',
+    'effective_status',
+)
+
+DEFAULT_CAMPAIGN_FIELDS = DEFAULT_FIELDS + (
+    'objective',
+    'start_time',
+    'stop_time',
+    'daily_budget',
+    'lifetime_budget'
+)
+
+DEFAULT_AD_FIELDS = DEFAULT_FIELDS + (
+    'adset_id',
+    'campaign_id'
```

Comment:
There is a missing comma between `campaign_id` and `creative`.

Reply:
huh! how does it work then :)